pyproject.toml

```diff
@@ -19,7 +19,8 @@ classifiers = [
 ]
 dependencies = [
     'numpy>=1.14.3',
-    'dpdata>=0.2.16',
+    'dpdata>=0.2.17',
```

Comment:
What's the reason to bump dpdata version?

Reply:
To better support cp2kdata with the newer dpdata version.

Reply:
it is ok to bump version here. I think any version of dpdata can support cp2kdata as long as the plugin function is implemented.

Reply:
The only concern is that cp2kdata still needs to be added to conda-forge. @robinzyb After I submit it, would you like to maintain it?

Reply:
yeah. I can maintain it. Maybe I can get strated from submitting it to conda-forge. I will do this tomorrow